honestcash-v2/src/app/core/models/post.ts

```diff
@@ -5,6 +5,7 @@ export type TPostTypeId = 'comment' | 'article' | 'response';
 export default class Post {
     id: number;
     title: string;
+    summary: string;
```

Comment:
For now I would not want any extensions of data model. Firstly we need to have a raw version of the editor with same functionality, later more.

Reply:
I will do completely one to one with the current working version with no summary, no image etc.